### ISSUE TITLE
the inline code `$` in tabs not show or render error

### ISSUE THREAD

**Issue by degbug** [jhildenbiddle/docsify-tabs#26]
this is my markdown content:
```
This is `$` (this dollar show)


<!-- tabs:start -->

#### **Tab A**

This is `$` (this dollar not show)

#### **Tab B**
This is `a` (this is show)
<!-- tabs:end -->
```

the render result is :

![image](https://user-images.githubusercontent.com/11486075/83534853-ef924380-a523-11ea-9973-cfbe68060f32.png)

the dollar symbol in the tabs was missed, how can i resolve this?

**Comment by jhildenbiddle** [comment]
Hey @degbug --

Apologies for the delayed response.

This is definitely a bug that I will look into, but if you're looking for a quick fix try one of the two options below:

1. Use HTML `<code>` tags instead of markdown backticks: `<code>$</code>`
1. Add a space after the dollar symbol: `` `$ ` ``

Neither option are ideal, but hopefully they will allow you to work around the issue until I can fix the issue with docsify-tabs.

**Comment by jhildenbiddle** [comment]
Fixed in 1.4.3
